README.md

```diff
@@ -324,17 +334,13 @@ Here are the Next.js app variables (e.g. in `cypress/dummy/.env`).
 # cypress/dummy/.env
 
 AUTH0_CLIENT_SECRET='FNo3i9f2fbFOdFH8f2fhsooi496bw4uGDif3oDd9fmsS18dDn'
-AUTH0_COOKIE_SECRET='DB208FHFQJFNNA28F0N1F8SBNF8B20FBA0BXSD29SSJAGSL12D9922929D'
-
-NEXT_PUBLIC_AUTH0_CLIENT_ID='FNfof292fnNFwveldfg9222rf'
-NEXT_PUBLIC_AUTH0_SCOPE='openid profile email'
-NEXT_PUBLIC_AUTH0_DOMAIN='lyft.auth0.com'
-NEXT_PUBLIC_AUTH0_REDIRECT_URI='http://localhost:3000/api/login-callback'
```

Comment:
You need to add "http://localhost:3000/api/auth/callback" to 'Allowed Callback URLs' in your tenant settings